org.kryogenix.Pick.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.kryogenix.Pick
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: pick-colour-picker
+
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+
+modules:
+  - name: pick-colour-picker
+    buildsystem: simple
+    build-commands:
+      - install -D pick-colour-picker-flatpak-runscript.sh /app/bin/pick-colour-picker
+      - chmod a+x /app/bin/pick-colour-picker
+      - cp -r pick /app/pick
+      - rm -rf /app/pick/__pycache__
+      - mkdir -p /app/share/metainfo/
+      - sed 's/pick-colour-picker.desktop/org.kryogenix.Pick.desktop/' pick-colour-picker.appdata.xml > /app/share/metainfo/org.kryogenix.Pick.appdata.xml
+      - install -D data/icons/scalable/apps/pick-colour-picker-symbolic.svg /app/share/icons/hicolor/scalable/apps/org.kryogenix.Pick.svg
+      - install -D data/icons/512x512/apps/pick-colour-picker.png /app/share/icons/hicolor/512x512/apps/org.kryogenix.Pick.png
+      - install -D data/icons/128x128/apps/pick-colour-picker.png /app/share/icons/hicolor/128x128/apps/org.kryogenix.Pick.png
+      - install -D data/icons/64x64/apps/pick-colour-picker.png /app/share/icons/hicolor/64x64/apps/org.kryogenix.Pick.png
+      - install -D data/icons/48x48/apps/pick-colour-picker.png /app/share/icons/hicolor/48x48/apps/org.kryogenix.Pick.png
+      - install -D data/icons/32x32/apps/pick-colour-picker.png /app/share/icons/hicolor/32x32/apps/org.kryogenix.Pick.png
+      - install -D data/icons/24x24/apps/pick-colour-picker.png /app/share/icons/hicolor/24x24/apps/org.kryogenix.Pick.png
+      - install -D data/icons/22x22/apps/pick-colour-picker.png /app/share/icons/hicolor/22x22/apps/org.kryogenix.Pick.png
+      - install -D data/icons/16x16/apps/pick-colour-picker.png /app/share/icons/hicolor/16x16/apps/org.kryogenix.Pick.png
+      - mkdir -p /app/share/applications/
+      - sed 's/Icon=.*/Icon=org.kryogenix.Pick/' pick-colour-picker.desktop > /app/share/applications/org.kryogenix.Pick.desktop
+    sources:
+      - type: git
+        url: https://github.com/stuartlangridge/ColourPicker.git
+        branch: app
```

Comment:
```suggestion
```
never branches.

Reply:
I don’t understand what this means? The app is in a git branch called “app”; that’s where it is. This isn’t a feature branch or anything; there isn’t a “main” or “master” branch.

Reply:
Yeah this could use a bit more explanation, but my understanding is if you're pointing to a specific commit, then specifying a branch isn't strictly necessary, at least.

Reply:
worse. branch get new commits which mean that the build will fail.

we want to reproduce build so it has to be a specific commit and if you specify a branch it will breach.

I personally wished we just made this a fatal error to use branches, but I have no control over this.

Reply:
Just so I understand this: I do currently specify a commit, and that’s fine, but I also specify a branch, and I should not do so; it’s not just irrelevant, as @cassidyjames  suggests, but it’s actually _wrong_ because newer pushes to that branch will get built and ignore the commit I specified?

Reply:
exactly what I'm saying.

ideally you'd specify a tag for a release, but I'm not hung on that part. just a commit is perfectly fine.

Reply:
I believe the below pushed update should resolve these issues, and thank you for the patience!